src/main/resources/org/jenkinsci/plugins/workflow/support/steps/stash/StashStep/help.html

```diff
@@ -1,5 +1,6 @@
 <div>
     Saves a set of files for use later in the same build, generally on another node/workspace.
+    The step will return a list of files that are included in the stash.
```

Comment:
File names, or paths? And if paths are they relative to the current working directory or workspace or absolute?